app/questionnaire/rules.py

```diff
@@ -281,3 +290,34 @@ def get_list_count(list_store, list_name):
 
 def is_goto_rule(rule):
     return any(key in rule.get('goto', {}) for key in ('when', 'block', 'group'))
+
+
+def get_default_answer(answer_id, answer_store, schema):
+    try:
+        block = schema.get_block_for_answer_id(answer_id)
+    except TypeError:
+        raise NoDefaultAnswerError
```

Comment:
What situation does this raise in? I'm not sure I like raising an error about default answers when there is a bigger problem if this happens. Isn't this more of a schema error?